tests/conftest.py

```diff
@@ -1946,6 +1946,18 @@ def cluster_teardown_finalizer():
                     kms.cleanup()
                 except Exception as ex:
                     log.error(f"Failed to cleanup KMS. Exception is: {ex}")
+            if ocsci_config.MULTICLUSTER.get("acm_cluster"):
+                try:
+                    from ocs_ci.utility.aws import AWS
+
+                    thanos_bucket_name = (
+                        f"dr-thanos-bucket-{config.ENV_DATA['cluster_name']}"
+                    )
+                    AWS().delete_bucket(thanos_bucket_name)
+                except Exception as ex:
```

Comment:
is there any specific exception that you can catch ?

Reply:
No idea, will have to fail the try block to see which exception is raised in this case.